mypy.ini

```diff
@@ -31,10 +31,12 @@ disable_error_code = attr-defined
 disable_error_code = import-not-found
 
 # - distutils._modified has different errors on Python 3.8 [import-untyped], on Python 3.9+ [import-not-found]
-# - All jaraco modules are still untyped
-# - _validate_project sometimes complains about trove_classifiers (#4296)
-# - wheel appears to be untyped
-[mypy-distutils._modified,jaraco.*,trove_classifiers,wheel.*]
```

Comment:
`trove_classifiers` was a symptom of the issue linked below https://github.com/python/mypy/issues/11936#issuecomment-1466764006 that is better worked around with the configuration below

Reply:
@FFY00